site/en/docs/extensions/mv3/hosting/index.md

```diff
@@ -1,69 +1,45 @@
 ---
 layout: "layouts/doc-post.njk"
-title: "Chrome Web Store"
+title: "Extension Hosting"
 date: 2012-09-18
-updated: 2018-06-12
+updated: 2021-12-10
 description: >
-  How to host your extension in the Chrome Web Store and update an extension
-  that's hosted in the Chrome Web Store.
+  How to host your Chrome extension.
```

Comment:
@dotproto I removed Chrome Web Store from the description. Is that okay? Any suggestion?